scripts/plot_exposure_diversity.R

```diff
@@ -0,0 +1,316 @@
+########################################
+#
+# PLOT: Diversity of exposure to news
```

Comment:
this was an old supplemental analysis. we don't really use it in the paper; however it might be useful to keep it for the review process. 

For now, I've cleaned it up a bit and moved it to the `suppl_code/` subdirectory.